### PR TITLE
[#2058] Added transitions properly for all of our plugins.

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2058-ensure-plugins-using-none-for-transitions-apply-display-appropriately

### CODE REVIEW

File: templates/assets/plugins/googlemap/popcorn.googlemap.js [hunk 1]
```diff
@@ -154,8 +153,11 @@ var googleCallback;
     newdiv.style.zIndex = +options.zindex;
     newdiv.style.position = "absolute";
 
-    if ( options.transition && options.transition !== "none" ) {
+    if ( options.transition !== "none" ) {
```
Comment (k88hudson) [inline templates/assets/plugins/googlemap/popcorn.googlemap.js]: IMO there's no reason to involve javascript here, your'e just adding lines. Have the default for`. off` = `visibility:hidden` with no transition.

In fact, isn't it already like that?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/9092fb4f9abd5a0/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/9092fb4f9abd5a0/output.txt

Total script time: 0.72 mins

Server is running on http://pr854.butyr.org/templates/basic/
Run the Test Suite: http://pr854.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/acedfc0c56f29d2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/acedfc0c56f29d2/output.txt

Total script time: 0.93 mins

Server is running on http://pr854.butyr.org/templates/basic/
Run the Test Suite: http://pr854.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/415ffc717969540/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/415ffc717969540/output.txt

Total script time: 0.90 mins

Server is running on http://pr854.butyr.org/templates/basic/
Run the Test Suite: http://pr854.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/d599473a3a2e1a2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d599473a3a2e1a2/output.txt

Total script time: 0.87 mins

Server is running on http://pr854.butyr.org/templates/basic/
Run the Test Suite: http://pr854.butyr.org/test/

File: templates/assets/plugins/titles/popcorn.titles.js [hunk 2]
```diff
@@ -76,7 +76,7 @@
         transition: {
           elem: "select",
           options: [ "None", "Pop", "Fade", ],
-          values: [ "none", "pop", "popcorn-fade" ],
+          values: [ "popcorn-none", "popcorn-pop", "popcorn-fade" ],
           label: "Transition",
```
Comment (k88hudson) [inline templates/assets/plugins/titles/popcorn.titles.js]: Why did you remove the other options?

Comment (mjschranz) [inline templates/assets/plugins/titles/popcorn.titles.js]: If memory serves me correct these are the transitions you wanted for titles. I think you felt that slide up/down wasn't appropriate for titles. If we want them in then that's an easy change.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0e28bb873909776/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0e28bb873909776/output.txt

Total script time: 0.81 mins

Server is running on http://pr854.butyr.org/templates/basic/
Run the Test Suite: http://pr854.butyr.org/test/

File: templates/assets/plugins/image/popcorn.image.css [hunk 3]
```diff
@@ -2,22 +2,19 @@ a.image-plugin-link {
...
 }
 
 div.image-plugin-container {
   overflow: hidden;
-  display: block;
   position: absolute;
 }
 
-div.image-plugin-hidden, a.image-plugin-hidden, img.image-plugin-hidden {
+a.image-plugin-hidden, img.image-plugin-hidden {
```
Comment (k88hudson) [inline templates/assets/plugins/image/popcorn.image.css]: This can be consolidated to `.image-plugin-hidden`.

File: templates/assets/plugins/titles/popcorn.titles.js (hunk 2 above)
Comment (k88hudson) [inline templates/assets/plugins/titles/popcorn.titles.js]: Please add them back in
